webapp/app/assessment/panEuropean/sections/table_1_4b/sectionSpec.js

```diff
@@ -0,0 +1,53 @@
+import * as PanEuropean from '@common/assessment/panEuropean'
+
+import * as SectionSpec from '@webapp/app/assessment/components/section/sectionSpec'
+
+const section = PanEuropean.sections['1'].children['14b']
+
+const variables = [
+  'harvested_wood_products',
+]
+
+const tableSpec = SectionSpec.newTableSpec({
+  [SectionSpec.KEYS_TABLE.name]: section.tables.table_1_4b,
+  [SectionSpec.KEYS_TABLE.columnsExport]: ['total_carbon_stock_in_hwp'],
+
+  [SectionSpec.KEYS_TABLE.rows]: [
+    // row header
+    SectionSpec.newRowHeader({
+      [SectionSpec.KEYS_ROW.cols]: [
+        SectionSpec.newColHeader({
+          [SectionSpec.KEYS_COL.labelKey]: 'panEuropean.carbonStockInHarvestedWoodProductsHWP.categoryYear',
+          [SectionSpec.KEYS_COL.left]: true,
+        }),
+        SectionSpec.newColHeader({
+          [SectionSpec.KEYS_COL.labelKey]: 'panEuropean.carbonStockInHarvestedWoodProductsHWP.total_carbon_stock_in_hwp',
+        }),
+      ],
+    }),
+
+    // rows data
+    ...variables.flatMap((variable) =>
+      PanEuropean.years90_15.reverse().map((year) =>
```

Comment:
this method PanEuropean.years90_15.reverse() update the original array and might create problems when accessing it again.
create a variable years on top of the file like we did in table 1.1a
```const years = [...PanEuropean.years90_15].reverse()

Reply:
The variable has been created in the '1.4b years variable created' commit.